src/include/util/dlist.h

```diff
@@ -144,6 +143,15 @@ static inline struct dlist_head *dlist_prev(const struct dlist_head *list) {
 	return list->prev;
 }
 
+/**
+ * @def dlist_head_init_entry()
+ *
+ * TODO
+ *
+ */
+#define dlist_head_init_entry(element, link_member) \
```

Comment:
Is this macro used anywhere?

Reply:
Was used, but now it's not. It needs to be removed, if not used anywhere, yes?